src/Compiler/Microsoft.AspNetCore.Razor.Language/src/SourceChange.cs

```diff
@@ -112,15 +110,15 @@ internal string GetOriginalText(SyntaxNode node)
         return node.GetContent().Substring(offset, Span.Length);
     }
 
-    public bool Equals(SourceChange other)
+    public bool Equals(SourceChange? other)
```

Comment:
💡 I would prefer to see the implemented inteface be `IEquatable<SourceChange?>` for clarity